11ty/definitions/dumb.md

```diff
@@ -8,18 +8,30 @@ defined: true
 excerpt: non-verbal, non-speaking, silent, refraining from speaking or unable to speak; sometimes __mute__ which can also be ableist; colloquially used to mean unintelligent or senseless
 speech: adjective
 alt_words:
-  - basic
-  - incomprehensible
-  - nonsensical
-  - redundant
-  - shallow
-  - silly
-  - senseless
-  - strange
-  - weird
-  - unfamiliar
-  - unfathomable
-  - unnecessary
+  dumb (noun):
+    - basic
+    - incomprehensible
+    - nonsensical
+    - redundant
+    - shallow
+    - silly
+    - senseless
+    - strange
+    - weird
+    - unfamiliar
+    - unfathomable
+    - unnecessary
+  dumbfounded (noun):
+    - amazed
+    - awed
+    - floored
+    - nonplussed
+  dumbstruck (noun):
+    - astonished
+    - perplexed
+    - shocked
+    - stunned
+    - stupified
```

Comment:
I think stupefied carries the same ableist loaded meaning as dumbfounded, right?

Reply:
Good catch!